### ISSUE TITLE
[Bug] Data panel crashed in kali linux 

### ISSUE THREAD

**Issue by MyatThu13** [cobbr/Covenant#264]
Data panel is crashed after running "Screenshot" Command

FormatException: The input is not a valid Base-64 string as it contains a non-base 64 character, more than two padding characters, or an illegal character among the padding characters.

System.Convert.FromBase64CharPtr(Char* inputPtr, int inputLength)

**Comment by on3rule2kill** [comment]
@MyatThu13 Good day, did you manage to solve this issue?

**Comment by Apr4h** [comment]
Hey there, I just came across the same issue while using Covenant. I found that I was able to recover from the Data panel crashing by deleting the records associated with the screenshot in Covenant/Covenant/Data/covenant.db.

My best guess is that it's crashing when trying to base64 decode the value of "FileContents" from the Events table, for the row associated with the screenshot, which contained message text from an Exception thrown while the grunt was trying to take the screenshot. If it helps, I took the screenshot from a 64bit .NET 4.0 Grunt running on Microsoft Windows NT 10.0.17134.0.

Happy to provide more help here if I can...
